src/RequestParser.cpp

```diff
@@ -574,8 +657,11 @@ void RequestParser::parseChunkedBody(HTTPRequest& request)
 			throw std::runtime_error(ERR_CHUNK_SIZE);
 		}
 		request.body += chunkData;
-		// length += numChunkSize;
+		length += numChunkSize;
 	} while (numChunkSize > 0);
+	std::stringstream sstream;
+	sstream << length;
+	request.headers["Content-Length"] = sstream.str();
```

Comment:
The contrary happens here: the conversion takes place from number to string in order to overwrite the value saved for "Content-Length" in the header map. Therefore the strstream was chosen.

Reply:
Yes I misread! 
In utilities there is a templated toString() function which could be used instead. Though I don't like it because it just seems heavy to me.

Reply:
Good hint, forgot about this one, replaced.